PocketKit/Sources/Analytics/Context/UIContext.swift

```diff
@@ -129,6 +132,9 @@ extension UIContext {
         public let myList = UIContext(type: .list, hierarchy: 0, identifier: .myList)
         public let archive = UIContext(type: .list, hierarchy: 0, identifier: .archive)
         public let favorites = UIContext(type: .list, hierarchy: 0, identifier: .favorites)
+        public let taggedChip = UIContext(type: .button, hierarchy: 0, identifier: .taggedChip)
+        public let notTagged = UIContext(type: .button, hierarchy: 0, identifier: .notTagged)
+        public let tagBadge = UIContext(type: .button, hierarchy: 0, identifier: .tagBadge)
```

Comment:
just preference, but we can also use `public static func button(identifier: Identifier) -> UIContext` instead of explicitly creating the context here.